CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Configure grafana-agent config.
```

Comment:
I would add a bit more to this to explain why we even add grafana-agent config, WDYT?